crypto/s2n_sequence.c

```diff
@@ -38,3 +42,17 @@ int s2n_increment_sequence_number(struct s2n_blob *sequence_number)
 
     return 0;
 }
+
+int s2n_sequence_number_to_uint64(struct s2n_blob *sequence_number, uint64_t *output)
+{
+    notnull_check(sequence_number);
+
+    uint8_t shift = 0;
+    *output = 0;
+
+    for (int i = sequence_number->size - 1; i >= 0; i--) {
```

Comment:
Do we have a coding standard on loop variables? is `int` what we use? Or would something like `size_t` make more sense?

Reply:
I'll change it to size_t